demos/js_thread_mode/README.md

```diff
@@ -1,16 +1,19 @@
-# js_thread_mode
+# JSRuntime Thread Mode
 
-js thread mode demo
+This Demo shows the JSRuntime management in WebF with different threading modes. There are three threading models:
```

Comment:
demo

Reply:
This demo illustrates the JSRuntime management in WebF with different threading modes. There are three threading models: